x11/urw-core35-fonts/Portfile

```diff
@@ -0,0 +1,61 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                urw-core35-fonts
+version             2017-07-27
+set git_version     91edd6e
+categories          x11 fonts
+maintainers         {@lemzwerg gnu.org:wl} openmaintainer
+license             AGPL-3-with-embedded-font-exception
```

Comment:
I don't know.  I wasn't able to find the list of licenses in either `macports-base` or `macports-ports`.  The tag `AGPL-3` is used a few times in ports, so this is certainly valid.  However, none of them is related to fonts...

Reply:
Sure, it is valid, but the `-with...` business is not. The `license` is used by automation, it is not a free input field.

Reply:
See the wiki about licenses. https://trac.macports.org/wiki/PortfileRecipes#license

Reply:
This isn't human documentation but intended to prevent incompatibly licensed software from being distributed by us. AGPL-3 will be fine I think, but see if that will conflict with licenses on packages that use these fonts.

Reply:
It's ok to specify a more restrictive licence, it's not ok to specify a more flexible one. If the added clause would add an exception to additionally limit the rights, it wouldn't be acceptable to write the original licence, but you may do so in this case.

In theory, perhaps nothing is wrong if you specify the licence in its current form, it's just that it will be treated as the most restrictive licence possible.

The worst thing that could happen is that MacPorts will refuse to distribute binaries for some ports which depend on these fonts due to licence conflict between `AGPL-3` and whatever other licence in the other port or dependency. It's usually possible to manually declare (override) a non-conflict of licences in individual ports.

An alternative would be to define a new licence in MacPorts and specify the matrix of (non)conflicts. I've never done that, so I'm not sure where this code resides. But if there's just one port using it, it sounds like an overkill.

Reply:
@mf2k, thanks for the list. Changed to AGPL 3.

Right now, no package uses this new port. Anyway, AGPL 3 is more permissive than GPL 2+ used in the old `urw-fonts` package; I thus don't see any potential conflicts.

Reply:
@mojca: Note that this exception is already part of ghostscript since many years!  The only difference is that it's somewhat hidden in ghostscript's `LICENSE` file, while I tried to make it explicitly visible.

Hopefully, we don't enter the nitpicking area...

Reply:
No need for nitpicking :)

Looks ok to me.